modules/developers-guide/pages/cli-reference/dfx-canister.adoc

```diff
@@ -64,14 +64,16 @@ For example, to register unique canister identifiers for a project when the {IC}
 dfx canister create --all
 ----
 
-If you want to register unique canister identifiers for the same project when the {IC} runs on the network that uses the `+ic+` alias in the `+dfx.json+` file, you can run the following command:
+If you want to register unique canister identifiers for the same project on the {platform}, you can run the following command:
 
 [source,bash]
 ----
 dfx canister --network ic create --all
 ----
 
-Similarly, you can call a canister and function running on a remote {IC} network using a command similar to the following:
+The {sdk-short-name} comes with an alias of `+ic+`, which is configured to point to the {platform}. You can also pass a URL as a network option, or you can configure additional aliases in `+dfx.json+` under the `+networks+` configuration.
+
+To illustrate, you can call a canister and function running on a remote {IC} network using a command similar to the following:
 
 [source,bash]
 ----
```

Comment:
omit "the replica" in L 103

Reply:
L209: 
Use the `+dfx canister create+` command to register one or more canister identifiers without compiled code on the {IC} network.
You must be connected to an {IC} network running locally or on a sub-network that you can access to run this command. 
=> 
Use the `+dfx canister create+` command to register one or more canister identifiers without compiled code.
You must be connected to a local canister execution environment or the Internet Computer blockchain to run this command.